### PR TITLE
Add org.sugarlabs.MakeThemFall

### PR DESCRIPTION
**Pull request by SudoSu-bham**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
   Make Them Fall, is a challenging distance game that tests your multitasking skills. Control up to 6 falling stickmen and avoid spikes and obstacles. Choose from six difficulty levels and see how long you can last. Play alone or with friends for added fun!

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by SudoSu-bham** [comment]
cc @tchx84

**Comment by SudoSu-bham** [comment]
bot, build org.sugarlabs.MakeThemFall

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "disable-external-data-checker": true
```
Comment (bbhtt) [inline flathub.json:2]: You disabled x-checker here yet you have a x-checker stanza in the manifest? That's not going to work if this is disabled here.

**Comment by bbhtt** [comment]
bot, build org.sugarlabs.MakeThemFall

**Pull request closed without merging**